sdk/storage/Azure.Storage.Blobs.Cryptography/src/ClientSideDecryptionPolicy.cs

```diff
@@ -0,0 +1,393 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Json;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.Cryptography;
+using Azure.Core.Http;
+using Azure.Core.Pipeline;
+using Metadata = System.Collections.Generic.IDictionary<string, string>;
+
+namespace Azure.Storage.Blobs.Specialized.Cryptography
```

Comment:
I understood we wanted to be under `Specialized` and also a `Cryptography` namespace. This is how I interpreted that.

While I agree it might be a little redundant to have one extra layer to the namespace, given the user already opting into the code by downloading the package in the first place, I also feel it's unlikely the code using this cryptography client will want anything to do with block, page, and append blob clients. Users probably don't want to clog intellisense with that when they just want encryption and don't care about the technical structure of azure storage. This extra layer gives them that option.

Reply:
Anyone using this feature is advanced enough that they can handle a few other clients in the namespace.  Also, the Arch Board already signed off on `.Specialized` and we'll need to go back to them (per https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming) if we want to change it.  I'd prefer `.Specialized`.